games/main.tex

```diff
@@ -293,34 +268,30 @@ \subsection{SWAP accounting with chequebook}
+A waiver essentially implements a proof that a cheque is destroyed and cannot be cashed, i.e., a promise that (part of) an uncashed cheque will not be redeemed.
+
+A \gloss{waiver} is implemented as a note signed by the peer holding uncashed cheques%. The note specifies the current \gloss{swap index} (sequential number) 
+and indicates the amount of debt they agree to waive from their entitlement.
+
+\subsubsection*{Cashing cheques in the context of waivers}
 If a channel is set to issue waived cheques then cashing must be a two step process.
-Upon receiving a cheque, the contract verifies the signature and checks if
-the swap index matches the one on the cheque.
-If the cheque is valid, the claim amount is stored in a variable with a timestamp.
-At this point a grace period starts: the original issuer gets notified of the cashing
-request and is invited to send in their last (highest) waiver signed by their peer.
-
-Analogously to cheques, waivers are accepted by the contract if sent in as data in
-a transaction together with the issuer's address.
-Upon receiving a waiver the contract verifies the signature and
-checks if the swap index matches the one in the waiver.
-If the waiver is valid, the peer's cumulative cashed balance is increased by the waived amount
-pretending the waived amount was paid out. Then the new cumulative cashed balance is
-compared to the one recorded when the cheque was received. Any remaining positive balance
-is transfered to the beneficiary, the cumulative sum is cleared from storage and
-the swap index is incremented.
+Upon receiving a cheque, the contract verifies the data, and %signature and checks if the swap index matches the one on the cheque.
+if the cheque is valid, the claimed amount is stored in a variable along with a timestamp. At this point a grace period starts: the original issuer gets notified of the cashing request and is invited to send in their last (highest) waiver signed by their peer.
+
+\subsubsection*{Cashing waivers}
+Analogously to cheques, waivers are accepted by the contract. %if sent in as data in a transaction together with the issuer's address.
+Upon receiving a waiver the contract verifies the data, and %signature and checks if the swap index matches the one in the waiver.
+if the waiver is valid, the peer's cumulative cashed balance is increased by the waived amount pretending the waived amount was paid out. Then the new cumulative cashed balance is compared to the one recorded when the cheque was received. Any remaining positive balance is transfered to the beneficiary, the cumulative sum is cleared from storage and the swap index is incremented.
+
+%%I stop here for now because the use of 'the swap index'  is confusing and I will rewrite it later ot be more clear.... a sequential index of waivers, stored by the checkbook separately from the serial number of the cheques but functioning like... how should this work? unclearly written
```

Comment:
actually, see fabio's comment, it is to be called swap cycle index, and is not indexing the waiver, just punctuates the periods when withheld cheques are used. 
Yes it is a little unclearly written here.